crates/re_viewer_context/src/space_view/visualizer_system.rs

```diff
@@ -66,6 +93,19 @@ pub trait VisualizerSystem: Send + Sync + 'static {
     }
 
     fn as_any(&self) -> &dyn std::any::Any;
+
+    /// Returns an initial value to use when creating an override for a component for this
+    /// visualizer. This is used as a fallback if the component doesn't already have data.
+    fn initial_override_value(
```

Comment:
Yeah, would love to avoid the overhead here. Type-erasure is such a pain. The generic UI also needs to be able to dispatch to it based on ComponentName. Would love to brainstorm some ways to make this path cleaner.